crates/core/tedge_agent/src/entity_manager/server.rs

```diff
@@ -270,6 +276,20 @@ impl EntityStoreServer {
         Ok(registered)
     }
 
+    async fn update_entity(
+        &mut self,
+        topic_id: &EntityTopicId,
+        new_parent: &EntityTopicId,
+    ) -> Result<&EntityMetadata, entity_store::Error> {
```

Comment:
Same as above, `update_parent` would be more appropriate

Reply:
Same as above. The scope of the update expanded beyond updating the parent.